package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "@anticrm/platform-root",
   "version": "0.4.0",
+  "engines": {
```

Comment:
Root package json is for some useful operations, it will not have any respect to main rush reactor.

Reply:
I totally agree with your point. It's just gentle reminder to users who will try it that the project requires Node.js 14.17+. Some modern IDEs already respect it and notify a developer about mismatch. 
See https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines

It's a minor thing. I can remove it

Reply:
No need to remove, if you interested please check if we could add same check for rush.

Reply:
> No need to remove, if you interested please check if we could add same check for rush.

Ok. I will